### PR TITLE
Fixed django multipart/form-data upload

### PR DESCRIPTION
**Pull request by evasseure**
## Description

This PR aims to fix an issue with django multipart/form-data uploads.

The current code only check if the content type is equal to `multipart/form-data`, but doesn't take into consideration that each item in a multipart message is separated by a boundary marker. For exemple webkit based browsers put "WebKitFormBoundary" in the name of that boundary.

So for webkit based browsers the content-type in the header would look like that: 
`Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryXQjBNYe84AdfPItt`

This PR simply changes the way we check the content type for `multipart/form-data`.

## Types of Changes

- [ ] Core
- [X] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Release type: minor
```
Comment (patrick91) [inline RELEASE.md:1]: I'd say this is a patch version 😊
```suggestion
Release type: patch
```

**Review by patrick91 — APPROVED** [review]
Awesome, didn't know about `Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryXQjBNYe84AdfPItt`

The change looks solid! I'll push a commit do this also for flask and ASGI :)

Feel free to fill this for to get a sticker! https://forms.gle/dmnfQUPoY5gZbVT67

**Comment by patrick91** [comment]
oh looks like only the django view had this problem, perfect then, will update the release note and merge 😊

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Fixed issue with django multipart/form-data uploads

**Pull request merged**